core/src/ldml/ldml_transforms.cpp

```diff
@@ -514,8 +537,21 @@ transforms::load(
       }
       transforms->addGroup(newGroup);
     } else if (group->type == LDML_TRAN_GROUP_TYPE_REORDER) {
-      // TODO-LDML: reorder
-      DebugLog("Skipping reorder (for now) TODO-LDML");
+      reorder_group newGroup;
+
+      // fetch each reorder in the group
+      for (KMX_DWORD itemNumber = 0; itemNumber < group->count; itemNumber++) {
+        const kmx::COMP_KMXPLUS_TRAN_REORDER *reorder = tranHelper.getReorder(group->index + itemNumber);
+
+        element_list elements;
+        element_list before;
+
+        bool load_ok = elements.load(kplus, reorder->elements) && before.load(kplus, reorder->before);
```

Comment:
From one of the failed build-checks:

```
07:46:48 
  [26/34] Compiling C++ object src/libkmnkbp0.so.0.0.0.p/ldml_ldml_transforms.cpp.o
07:46:48 
  FAILED: src/libkmnkbp0.so.0.0.0.p/ldml_ldml_transforms.cpp.o
07:46:48 
  c++ -Isrc/libkmnkbp0.so.0.0.0.p -Isrc -I../../../src -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Werror -std=c++14 -O3 -fPIC -isystem../../../../common/include -isystem../../../include -isysteminclude -DKMN_KBP -DKMN_KBP_EXPORTING -Wctor-dtor-privacy -Wdouble-promotion -Wendif-labels -Wno-unknown-pragmas -Wno-missing-field-initializers -Wnon-virtual-dtor -Wshadow -fvisibility=hidden -fvisibility-inlines-hidden -MD -MQ src/libkmnkbp0.so.0.0.0.p/ldml_ldml_transforms.cpp.o -MF src/libkmnkbp0.so.0.0.0.p/ldml_ldml_transforms.cpp.o.d -o src/libkmnkbp0.so.0.0.0.p/ldml_ldml_transforms.cpp.o -c ../../../src/ldml/ldml_transforms.cpp
07:46:48 
  ../../../src/ldml/ldml_transforms.cpp: In static member function ‘static km::kbp::ldml::transforms* km::kbp::ldml::transforms::load(const km::kbp::kmx::kmx_plus&, const km::kbp::kmx::COMP_KMXPLUS_TRAN*, const km::kbp::kmx::COMP_KMXPLUS_TRAN_Helper&)’:
07:46:48 
  ../../../src/ldml/ldml_transforms.cpp:549:14: error: unused variable ‘load_ok’ [-Werror=unused-variable]
07:46:48 
    549 |         bool load_ok = elements.load(kplus, reorder->elements) && before.load(kplus, reorder->before);
07:46:48 
        |              ^~~~~~~
07:46:48 
  cc1plus: all warnings being treated as errors
```

Note the final three log-lines, in particular.

Reply:
Unused because of the assert, right. Will fix.